deployments/helm/KubeArmorOperator/Chart.yaml

```diff
@@ -13,9 +13,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: v0.10.2
```

Comment:
Folks, can you please confirm if this versioning is right?
It will be the same as KubeArmor.
cc @rksharma95 @daemon1024

Reply:
Also, I'm not upto speed with how have we decided to publish charts for managed platforms (which need certification) or have we kept it as a discussion for v0.12.

Reply:
imo we should be fine with versioning charts same as KubeArmor. and yeah for other platforms we'll need to have workkflows for both helm charts and container images publishing. we'll target it in v0.12.

Reply:
Cool. : )